### PR TITLE
Working preview!

### PR DESCRIPTION
**Pull request by alexgirard**
- made a bunch of more temporary changes in order to get the preview working
- small ui updates

Screen Shot 2021-06-22 at 12 38 25 AM

### CODE REVIEW

File: src/student/MultipleChoicePage.tsx [hunk 1]
```diff
@@ -1,21 +1,50 @@
...
+  { identifier: 'D' },
+] as Answer[];
+
+export const MultipleChoicePage = ({
+  question,
+  answers = defaultAnswers,
+}: Props) => {
   return (
     <StudentPageBase>
+      {question && (
```
Comment (alexgirard) [inline src/student/MultipleChoicePage.tsx:26]: added here for now, we should make a wrapper component including. the title for all question types in. student later

File: src/theme.ts [hunk 2]
```diff
@@ -14,7 +14,11 @@ const blues = [
   '#E6E8EF', // instructor pale blue
   '#214DC230', // instructor translucent blue
   '#214DC295', // instructor almost opaque blue
-  '#AE6FFF', // royal blue
+];
+
+// eslint-disable-next-line
+const purples = [
```
Comment (alexgirard) [inline src/theme.ts:20]: not sure why royal blue before -> was actually purple

File: src/instructor/CreateQuiz.tsx [hunk 3]
```diff
@@ -8,7 +8,7 @@ import {
   Question,
   QuestionFormat,
   QuestionType,
-} from 'src/instructor/questions/Types';
+} from 'src/components/Types';
```
Comment (heathermusson) [inline src/instructor/CreateQuiz.tsx:11]: Why did this change?

**Review by heathermusson — APPROVED** [review]

File: src/theme.ts (hunk 2 above)
Comment (BenLanglois) [reply src/theme.ts:19]: Probably because I named it 😅

File: src/instructor/CreateQuiz.tsx (hunk 3 above)
Comment (alexgirard) [reply src/instructor/CreateQuiz.tsx:11]: since its now shared with the student side, I changed it to no longer live under "instructor"

**Pull request merged**